resolwe/flow/tests/test_executors.py

```diff
@@ -232,6 +232,7 @@ def test_cpu_resource(self):
         self.run_process('test-cpu-resource-1core')
         self.run_process('test-cpu-resource-2core')
 
+    @unittest.skipIf(settings.FLOW_DOCKER_DISABLE_SECCOMP, "Docker seccomp is disabled.")
```

Comment:
Why?

Reply:
Because it doesn't make sense to test seccomp if it is disabled :)